.ci/common.sh

```diff
@@ -48,7 +50,16 @@ check_core_dumps() {
 }
 
 setup_deps() {
+	sudo pip install --upgrade pip
 	sudo pip install neovim
+
+	# For pip3
+	# https://github.com/travis-ci/travis-ci/issues/1528
+	# sudo apt-get install -q python3.3-dev
+	# curl -Ss http://python-distribute.org/distribute_setup.py | sudo python3
+	# curl -Ss https://raw.github.com/pypa/pip/master/contrib/get-pip.py | sudo python3
+	# sudo pip3.3 install neovim
```

Comment:
Do you have a link to the most recent and relevant build failure?  I'd like to take a look into it.